### PR TITLE
feat: Add backfill notice

### PR DESCRIPTION
**Pull request by lynnagara**
image

### CODE REVIEW

**Comment by lynnagara** [comment]
screen shot 2018-05-02 at 3 59 52 pm

**Comment by Chrissy** [comment]
lol best description

**Comment by Chrissy** [comment]
This now uses cookies and needs review from a javascript person!

Styles can be reviewed by 🤷‍♀️ probably @ckj.

File: src/sentry/static/sentry/app/components/projectHeader/backfillNotice.jsx [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+  };
+
+  render() {
+    const shouldDisplayWarning =
+      moment(BACKFILL_DATE).isAfter(moment(this.props.project.dateCreated)) &&
+      Cookies.get('backfill_notification_closed') !== 'true' &&
+      this.state.isDismissed == false;
+
+    return (
+      shouldDisplayWarning && (
```
Comment (billyvg) [inline src/sentry/static/sentry/app/components/projectHeader/backfillNotice.jsx]: Can we explicitly return `null` if this is false to avoid extra nesting?

**Review by billyvg — APPROVED** [review]

**Comment by Chrissy** [comment]
ok @benvinegar @mattrobenolt sorry about that. I think Lyn was in a hurry when she put that part together. I'll change it to say "Some Data before April 2nd may be unavailable"

**Comment by Chrissy** [comment]
cc @ckj design is updated

**Review by ckj — APPROVED** [review]
Like the updates. lgtm!

**Pull request merged**